app/editor/src/features/admin/sources/DataSourceMetrics.tsx

```diff
@@ -0,0 +1,51 @@
+import { Row } from 'components/flex';
+import { FieldSize } from 'components/form';
+import { FormikText } from 'components/formik';
+import { getIn, useFormikContext } from 'formik';
+import { useNamespace } from 'hooks';
+import { IDataSourceModel, ISourceMetricModel } from 'hooks/api-editor';
+import React from 'react';
+import { useLookup } from 'store/hooks';
+
+import * as styled from './styled';
+
+export const DataSourceMetrics: React.FC = (props) => {
```

Comment:
Essentially the same thing as the `DataSourceActions`.  I should make a generic component that handles different types of objects.  This component is also very slow at displaying results regrettably.